src/zcl_dirigible_employee_dao.clas.abap

```diff
@@ -1,24 +1,29 @@
 CLASS zcl_dirigible_employee_dao DEFINITION PUBLIC.
+
   PUBLIC SECTION.
-    TYPES: BEGIN OF ty_employee,
-             id         TYPE n LENGTH 10,
-             firstName  TYPE string,
-             lastName   TYPE string,
-           END OF ty_employee.
 
-    TYPES: ty_employees TYPE TABLE OF ty_employee WITH EMPTY KEY.
+    TYPES:
+      BEGIN OF ty_employee,
+        id         TYPE n LENGTH 10,
+        firstName  TYPE string,
+        lastName   TYPE string,
+      END OF ty_employee,
+      ty_employees TYPE TABLE OF ty_employee WITH EMPTY KEY.
+
+    CLASS-METHODS:
+      select_all
+        RETURNING VALUE(rv_result) TYPE string.
 
-    CLASS-METHODS: select_all_employees
-      RETURNING VALUE(rv_result) TYPE string.
 ENDCLASS.
 
 CLASS zcl_dirigible_employee_dao IMPLEMENTATION.
-  METHOD select_all_employees.
+
+  METHOD select_all.
     DATA: lt_employees TYPE ty_employees,
           ls_employee  TYPE ty_employee,
           lv_result    TYPE string.
 
-    SELECT *
+    SELECT id, firstName, lastName 
```

Comment:
```suggestion
    SELECT id firstName lastName 
```

commas are required in strict mode ABAP SELECTS, which is not possible on 702, the transpiler only takes 702 input